br/pkg/task/common.go

```diff
@@ -92,6 +92,9 @@ const (
 	flagCipherKey     = "crypter.key"
 	flagCipherKeyFile = "crypter.key-file"
 
+	flagMetadataDownloadBatchSize    = "metadata-download-batch-size"
+	defaultMetadataDownloadBatchSize = 64
```

Comment:
will it makes restore slow by default? maybe we can set it to 128 to keep the compatibility